model_logging/migrations/0003_data_migration.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+
+app = 'model_logging'
+model = 'LogEntry'
+
+
+def move_data(apps, schema_editor):
+    try:
+        from pgcrypto.fields import TextPGPPublicKeyField
+    except ImportError:
+        raise ImportError('Please install django-pgcrypto-fields to perform migration')
```

Comment:
I think you can make this easier by providing a squash migration as well (which will cover all the existing migrations and be run instead if the database is empty).